sys/include/net/gcoap.h

```diff
@@ -567,9 +614,14 @@ extern "C" {
  * @brief Stack size for module thread
  */
 #ifndef GCOAP_STACK_SIZE
+#if IS_ACTIVE(CONFIG_GCOAP_ENABLE_DTLS)
+#define GCOAP_STACK_SIZE (2*THREAD_STACKSIZE_DEFAULT + DEBUG_EXTRA_STACKSIZE \
```

Comment:
let's rather define a `DTLS_EXTRA_STACKSIZE` that gets added here

Reply:
Done. I always define `GCOAP_DTLS_EXTRA_STACKSIZE`, but set it to 0 for non-DTLS.